docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/vimeo-tracking/index.md

```diff
@@ -0,0 +1,207 @@
+To illustrate the tracked events and entities, you can visit an example app that showcases the tracked media events and entities live as you watch a video.
+
+There are examples for both the [iframe](https://snowplow-incubator.github.io/snowplow-javascript-tracker-examples/vimeo) and [player](https://snowplow-incubator.github.io/snowplow-javascript-tracker-examples/vimeoPlayer) methods of tracking a Vimeo video.
+
+Source code for the app is [available here.](https://github.com/snowplow-incubator/snowplow-javascript-tracker-examples)
+:::
+
+## Basic Usage
+
+The snippets below show how to get started with the plugin, after [setting up your tracker](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/tracker-setup/installing-the-tracker-from-npm/index.md).
+
+:::info The plugin's `video` attribute will accept either:
+
+- An `iframe` element
+- An instance of `Vimeo.Player`, created with the [Vimeo Player SDK](https://developer.vimeo.com/player/sdk)
+
+:::
+
+<Tabs>
+  <TabItem value='iframe' label='Iframe' default>
+
+```html
+<iframe
+  id='vimeo-iframe'
+  src='https://player.vimeo.com/video/535907279?h=db7ea8b89c'
+></iframe>
+```
+
+```javascript
+import { startVimeoTracking, endVimeoTracking } from '@snowplow/browser-plugin-vimeo-tracking'
+
+const id = 'XXXXX'; // randomly generated ID
+const video = document.getElementById('vimeo-iframe')
+
+startVimeoTracking({ id, video })
+
+// Vimeo events are tracked...
+
+endVimeoTracking(id)
+```
+
+</TabItem>
+<TabItem value='player' label='Vimeo.Player'>
+
+```html
+<div id='vimeo-player'></div>
+```
+
+```javascript
+import { Player } from '@vimeo/player'
+import { startVimeoTracking, endVimeoTracking } from '@snowplow/browser-plugin-vimeo-tracking'
+
+const id = 'XXXXX'; // randomly generated ID
+const video = new Player('vimeo-player', {
+  videoId: 'zSM4ZyVe8xs'
+});
+
+startVimeoTracking({ id, video })
+
+// Vimeo events are tracked...
+
+endVimeoTracking(id)
+```
+
+  </TabItem>
+</Tabs>
+
+:::caution
+
+It's important to call `endVimeoTracking` as this will end any recurring ping events, clear all listeners set by the Vimeo plugin, along with resetting statistics counters used by the Snowplow Media plugin.
+
+:::
+
+## Selecting events to track
+
+<details>
+<summary>The plugin provides automatic tracking of the following events</summary>
+
+| Vimeo Event Name               | string                              | Description                                                                                                           |
+| ------------------------------ | ----------------------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| Ready                          | 'ready'                             | Media player event fired when the media tracking is successfully attached to the player and can track events.         |
```

Comment:
I think there is no need to prefix every single line with “Media player event”. Also is there any semantic difference between “fired”, “sent” and “tracked”? If not, I would use a single verb to avoid confusion